makedist.pl

```diff
@@ -84,8 +88,17 @@
 		@mlist = @mlist_excluded;
 		}
 	}
-@dirlist = ( "vendor_perl" );
 
+# Build EOL data
```

Comment:
What's this code doing here? I don't know if it's a good idea to have makedist.pl download the EOL file..

Reply:
This code builds/updates `os_eol.json` upon Webmin build. Otherwise, we'd have to remember to do it manually.

Reply:
But isn't this file just downloaded from the API we call?

Reply:
Yes, I gave up on the idea of updating this file on the user side. Instead, we will download it at Webmin build time and locally provide this file for Webmin users.

Reply:
Ah that makes sense. But in that case, do we still need all the code that in os-eol-lib.pl that checks for cache times?

Reply:
Yes, we do. It's new code by the way, and it's crucial so we don't read and parse 31 kB `os_eol.json` file everytime! It makes things much more efficient.

Reply:
So is that gets included in the package just the JSON, or does it get parsed into some other format first before packaging?

Reply:
No, it will be just a single `os_eol.json` file that we will include into Webmin build and update it everytime new Webmin is out.

Reply:
What is the `eol_update_cache` function in this PR needed for then?

Reply:
To extract a few bytes of EOL-related data for the _current_ OS, rather than dealing with hundreds of systems (from `os_eol.json` file) all the time.

Reply:
What I mean is, why is there a need to check any cache time here : 

![image](https://github.com/webmin/webmin/assets/102706/48c391c5-e444-47eb-a771-8a6a1676273a)

Reply:
> What I mean is, why is there a need to check any cache time here

Do you mean why not just do it once, after Webmin was upgraded?

Reply:
Actually, I [just tweaked](https://github.com/webmin/webmin/commit/0b21cbdc7c7a2f32c13552b63e8efb1f34823d7a) it a bit. Considering that the EOL database can only be changed during a Webmin update, we don't need to perform checks every 30 days!

Reply:
Thanks, that looks more reasonable!